05.wc/wc.rb

```diff
@@ -0,0 +1,71 @@
+# ! /usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  argv_options = parse_and_remove_options(ARGV)
+  output_text = []
+  ARGF.each(nil) do |input_text|
+    wc_info = []
```

Comment:
最後に to_h するのではなく、最初からハッシュとして扱う方が自然だと思います
```suggestion
    wc_info = {}
```